oauthproxy_test.go

```diff
@@ -2938,3 +2938,57 @@ func TestAuthOnlyAllowedGroups(t *testing.T) {
 		})
 	}
 }
+
+func TestAuthOnlyRedirectSignIn(t *testing.T) {
+	testCases := []struct {
+		name               string
+		querystring        string
+		expectedStatusCode int
+	}{
+		{
+			name:               "RedirectSignInTrueSignInPageCalledWithInvalidToken",
+			querystring:        "?redirect_signin=true",
+			expectedStatusCode: http.StatusForbidden,
+		},
```

Comment:
Why does this end up as a forbidden response? Do we have a test case for a successfuly redirect_signin, ie it would render a page and return and ok status?

Reply:
I did not know how to pass a valid token for the `SignInPage` call to succeed, so made the test name indicate an invalid token is being passed. I can have a look to add a successful render test case but would appreciate if you could point to an existing test that passes a valid token, will certainly make it easier for me

Reply:
With the redirect option, the test checks for `302` but will add more tests if there is an agreement on the approach please